core/ui/EditTemplate/fields.tid

```diff
@@ -71,6 +71,7 @@ $value={{{ [<newFieldValueTiddler>get[text]] }}}/>
 <em class="tc-edit">
 <<lingo Fields/Add/Prompt>>&nbsp;&nbsp;
 </em>
+<div class="tc-edit-field-add-name-wrapper">
 <span class="tc-edit-field-add-name">
```

Comment:
is the span obsolet after adding a wrapper? imo it should be

Reply:
thanks @pmario , yes it is, I've removed it